Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -49,17 +49,68 @@ public override string ToString()
     [IonDoNotAnnotateType(ExcludeDescendants = true)]
     public class Motorcycle : Vehicle
     {
-        public string Brand { get; init; }
+        public string Make { get; init; }
+
+        [IonField]
+        public string color;
+
+        [IonField]
+        public bool canOffroad;
+        
+        public override string ToString()
+        {
+            return $"<Motorcycle>{{ Make: {this.Make}, color: {this.color}, canOffroad: {this.canOffroad} }}";
+        }
+    }
+
+    // For testing case insensitive deserialization.
+    public class MOTORCYCLE : Vehicle
+    {
+        public string MAKE { get; init; }
+
+        [IonField]
+        public string COLOR;
+
+        [IonField]
+        public bool CANOFFROAD;
+        
+        public override string ToString()
+        {
+            return $"<MOTORCYCLE>{{ MAKE: {this.MAKE}, COLOR: {this.COLOR}, CANOFFROAD: {this.CANOFFROAD} }}";
+        }
+    }
+    
+    // For testing case insensitive deserialization.
+    public class motorcycle : Vehicle
```

Comment:
You don't need these types.

Reply:
Do you have something different in mind for how to test this other than defining classes with the same property names (but with different casing)?